relay-metrics/src/meta/protocol.rs

```diff
@@ -50,13 +50,13 @@ pub struct Location {
     pub lineno: Option<u64>,
     /// Source code leading up to `lineno`.
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub pre_context: Vec<String>,
+    pub pre_context: Vec<Option<String>>,
     /// Source code of the current line (`lineno`).
     #[serde(skip_serializing_if = "Option::is_none")]
     pub context_line: Option<String>,
     /// Source code of the lines after `lineno`.
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub post_context: Vec<String>,
+    pub post_context: Vec<Option<String>>,
```

Comment:
Should we leave this as `Vec<String>` and filter out `None` values on deserialization? Or do the `None` values actually have significance / help the user?